publish/2019-10-20-WhatsNew020.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "What's New in Release 0.2.0"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/50876789
+author_github: https://github.com/stephenkinder
+seo-title: What is new in release 0.2.0
+seo-description: What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides.
+blog_description: "What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides."
+---
+
+= What's new in 0.2.0
+Stephen Kinder <https://github.com/stephenkinder>
+
+Kabanero Foundation is a new open source project focused on bringing together foundational open source technologies into a modern microservices-based framework.   With the first update of the Kabanero project, we've been busy making a ton of improvements.  
+
+Control
+The Kabanero operator introduces the `--targetNameSpace` parameter, which defines the set of valid NameSpaces for which the Appsody operator watches for the deployment of applications.  
+
+Ease of Use
+The Kabanero operator has introduced variable substitution in the collection pipelines, simplifying collection curation.   The Kabanero Foundation provides a must-gather script to help diagnostic activities, and an uninstall script which can help in try-it and POC activities.
+
+Kabanero installs an OKD console extension we've called the OKD landing page.   The Landing Page has been enhanced to display active collections, links to supporting software, like the Tekton dashboard, as well as delivering documentation and new Guides accessible right from your OKD cluster.
+
+image::/img/guide/Kabanero020.png[link="/img/Kabanero020.png" alt="Kabanero 0.2.0"]
```

Comment:
this is the first img we're adding to blogs, so the below image file should go in a folder `img` at the root of the repo. Then both links would be `/img/blog/Kabanero020`

Reply:
I've updated to put the png file in /img/blog/